.secrets.baseline

```diff
@@ -414,7 +477,7 @@
       }
     ]
   },
-  "version": "0.13.1+ibm.51.dss",
+  "version": "0.13.1+ibm.47.dss",
```

Comment:
@gayathrimenath-ibm Please upgrade `detect-secrets` tool and regenerate this file
`pre-commit autoupdate`
https://github.ibm.com/Whitewater/whitewater-detect-secrets/wiki/Developer-Tool-FAQs#how-do-i-upgrade-detect-secrets-to-a-newer-version